Milestone4

amita-roy opened the pull request:
## Step Completed in this milestone 

### Implement the calculate.js module

- I have created a calculate function.

- My Calculate function is accepting 2 parameters (data and buttonName)

- Data object in the parameter contains three properties (total, next, and operation)

- I have created a"buttonOperation" function separately to do all the calculations but in the same module, and my calculate method is calling the "buttonOperation" function and passing all the required parameters to it, and "buttonOperation" will return the result.


### Implement the operate.js module

- This module is importing the big.js module from the corresponding npm package.

- I have created a function named operate.

- The operate function is accepting num1, num2, and operation as arguments.

- Using the value of the operation argument, operate function is performing a simple math operation.

- I have wrapped the number parameter with Big() to work with big numbers

amita-roy committed:
Update readme for milestone4

amita-roy committed:
Add calculate.js and operate.js file in logic folder

amita-roy committed:
Add operate and calculate functionality

amita-roy committed:
Add renderButton through iteration

amita-roy committed:
Logg each button name on console

amita-roy committed:
Update app class

amita-roy committed:
Update Calculate js file

amita-roy committed:
Update readme with step completed in this milestone

addod19 reviewed (changes requested):
## Status: Require changes :recycle: :traffic_light: 

Hi @Amita-Roy, congratulations on this project :smile: The PR has a meaningful title and a brief description :+1: However, let us kindly check the comments below so we can get our project approved:
- The README file needs to be improved so we can have instructions about how to clone and run the project, live link, authors social handles, etc
- `calculate.js` should be imported in the app component, currently, it is not
- `calculate.js module` has some missing requirements
   - It should contain only one function, but we currently have 2 functions inside our calculate.js file
- `operate.js module` some of the requirements has not been met, kindly check comments below
   - `numberOne` and `numberTwo` are wrapped with the Big class, currently, we have only one of the variables wrapped with the Big class
   - Kindly use the `modifier` instead of the `div(100)` used currently

- I however noticed that milestone 3 has not been implemented, kindly fix the styling before proceeding

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

amita-roy committed:
Wrapped num2 with big

amita-roy committed:
Update the operate js functionality

amita-roy committed:
Update the readme as suggested

amita-roy committed:
Add live link in readme

amita-roy committed:
Import calculate js in App

Amita-Roy commented:
Implemented the readme as suggested.

-  I have imported the calculate.js in the App component as per the suggestion but at this point of project we are not using calculate.js in the App so es-lint is giving some error for no-unsed-vars so I have commented it out.

- In the calculate js requirements, it said that it should have one default method: calculate, which I have but it not mentioned that you cannot use another function to breakdown the functionality.  In any case, I have calculate functioning which getting exported as default, not others. Please don’t refer to the below screenshot.

- Wrapped both the number with Big() as suggested

- Changed the div(100) to modifier

- Milestone3 was just for submission, It suggested not to implement the design right now because they are going to ask in further sections. Please refer to the below screenshot.

<img width="594" alt="Screenshot 2021-03-18 at 9 47 22 AM" src="https://user-images.githubusercontent.com/47393794/111598857-eed74f00-87cf-11eb-9098-d7252776f32f.png">


<img width="934" alt="Screenshot 2021-03-17 at 8 51 23 PM" src="https://user-images.githubusercontent.com/47393794/111598873-f39c0300-87cf-11eb-84d6-7d2fb208dfca.png">

mikenath223 reviewed (approved):
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**
